service/frontend/workflow_handler.go

```diff
@@ -1955,6 +1939,9 @@ func (wh *WorkflowHandler) SignalWorkflowExecution(ctx context.Context, request
 		return nil, err
 	}
 
+	metricsHandler := wh.metricsScope(ctx).WithTags(metrics.CommandTypeTag(enumspb.COMMAND_TYPE_UNSPECIFIED.String()))
```

Comment:
Ideally we would know what API this metric was emitted from. I would use a single label that can be applied to both a command and an API.